### PR TITLE
Mauricio z/Squarelotron

### PR DESCRIPTION
**Pull request by mauricioZelaya**
uploading final version of squarelotron homework

### CODE REVIEW

File: src/main/java/org/minions/devfund/mauricioz/Squarelotron.java [hunk 1]
```diff
@@ -0,0 +1,193 @@
+package org.minions.devfund.mauricioz;
+/**
+ * Squareotron class.
+ */
+public class Squarelotron {
+
+    private int[][] squarelotron;
```
Comment (carledriss) [inline src/main/java/org/minions/devfund/mauricioz/Squarelotron.java]: MAJOR Rename field "squarelotron" [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1700)

**Comment by carledriss** [comment]
SonarQube analysis reported 1 issue
* ![MAJOR][MAJOR] 1 major

Watch the comments in this conversation to review them.

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Comment by mauricioZelaya** [comment]
image

image

**Review by carledriss — CHANGES_REQUESTED** [review]
@mauricioZelaya 
Please update against develop

**Pull request closed without merging**